docs/adr/003-api-versioning.md

```diff
@@ -0,0 +1,28 @@
+# ADR 003 - API versioning
+
+Date: 26-11-2024
+
+## Status
+
+Accepted
+
+Written years after the decision was made.
+
+## Context
+
+By versioning the API we can communicate changes in the API. If an endpoint changes in a way that breaks backwards
+compatibility we will change the route version. An example of a backwards compatibility break is changing field names.
+
```

Comment:
Is it necessary to say that we use sem or simver?

Reply:
We do not use semver for routes. We use /v1, /v2, etc prefix for routes.